<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Carla_Simulator</class>
 <widget class="QDialog" name="Carla_Simulator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>378</width>
    <height>541</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Carla_Simulator</string>
  </property>
  <widget class="QTabWidget" name="Simulator">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>361</width>
     <height>521</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Setting</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>15</y>
       <width>331</width>
       <height>471</height>
      </rect>
     </property>
     <property name="title">
      <string>차량 목록</string>
     </property>
     <widget class="QPushButton" name="pushButton_remove_Vehicle">
      <property name="geometry">
       <rect>
        <x>182</x>
        <y>428</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>초기화</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_generate_Vehicle">
      <property name="geometry">
       <rect>
        <x>38</x>
        <y>428</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>경로 설정</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="VehicleTextBrowser">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>26</y>
        <width>291</width>
        <height>391</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Location</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>15</y>
       <width>331</width>
       <height>471</height>
      </rect>
     </property>
     <property name="title">
      <string>차량 위치</string>
     </property>
     <widget class="QTextBrowser" name="LocationTextBrowser">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>26</y>
        <width>291</width>
        <height>401</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="LocationComboBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>435</y>
        <width>291</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_5">
    <attribute name="title">
     <string>Traffic sign</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>15</y>
       <width>331</width>
       <height>471</height>
      </rect>
     </property>
     <property name="title">
      <string>Traffic Sign Data</string>
     </property>
     <widget class="QPushButton" name="pushButton_add_Sign">
      <property name="geometry">
       <rect>
        <x>26</x>
        <y>426</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>추가</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="trafficSignTextBrowser">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>26</y>
        <width>291</width>
        <height>391</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_initialize_Sign">
      <property name="geometry">
       <rect>
        <x>124</x>
        <y>426</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>초기화</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_apply_Sign">
      <property name="geometry">
       <rect>
        <x>224</x>
        <y>426</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>적용</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Configuration</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_Time">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>106</y>
       <width>341</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial Narrow</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Time_Control</string>
     </property>
     <widget class="QRadioButton" name="radioButton_1">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>90</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial Narrow</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Midday</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_2">
      <property name="geometry">
       <rect>
        <x>192</x>
        <y>30</y>
        <width>90</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial Narrow</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Sunset</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_3">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>60</y>
        <width>90</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial Narrow</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Midnight</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_NPC">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>7</y>
       <width>341</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial Narrow</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="title">
      <string>NPC_Spawn</string>
     </property>
     <widget class="QLineEdit" name="Edit_People">
      <property name="geometry">
       <rect>
        <x>93</x>
        <y>26</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Edit_Vehicle">
      <property name="geometry">
       <rect>
        <x>93</x>
        <y>56</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_NPC_Spawn">
      <property name="geometry">
       <rect>
        <x>233</x>
        <y>26</y>
        <width>71</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial Narrow</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Replace</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>22</x>
        <y>28</y>
        <width>56</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial Narrow</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Walkers</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>22</x>
        <y>56</y>
        <width>56</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial Narrow</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Vehicles</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_Weather">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>341</width>
       <height>171</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial Narrow</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Weather_Control</string>
     </property>
     <widget class="QSlider" name="horizontalSlider_fog">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>31</y>
        <width>160</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="horizontalSlider_rain">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>61</y>
        <width>160</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="horizontalSlider_wind">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>91</y>
        <width>160</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="horizontalSlider_clouds">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>121</y>
        <width>160</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>31</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial Narrow</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Fog</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>61</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial Narrow</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Rain</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>91</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial Narrow</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Wind</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>121</y>
        <width>56</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial Narrow</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Clouds</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
